app/templates/main/browse-consignment.html

```diff
@@ -99,340 +71,247 @@
+                                </div>
+                                <div class="govuk-radios__item govuk-radios__item--consignment">
+                                    <input class="govuk-radios__input" id="recordStatus-all" name="record_status" type="radio"
+                                        value="all" {% if user_filters["record_status"]=='all' %}checked{% endif %} {% if not
+                                        user_filters["record_status"] %}checked{% endif %}>
+                                        <label class="govuk-label govuk-radios__label govuk-radios__label--consignment"
+                                               for="recordStatus-all">All records</label>
+                                    </div>
+                                </div>
+                            </fieldset>
                         </div>
                     </div>
-                    <div class="filters-form__series__container">
-                        <p class="govuk-body browse__body">Series</p>
-                        <div class="govuk-form-group filters-form__group filters-form__series-group">
-                            <label class="govuk-label" for="browse_filter_series"></label>
-                            <input class="govuk-input filters-form__series--input"
-                                   id="browse_filter_series"
-                                   name="width10"
-                                   type="text">
+                    <!-- FILE TYPE -->
+                    <div class="consignment-filter-container consignment-filter-container--file-type">
+                        <div class="file-type-doc--container">
+                            <h3 class="govuk-heading-s govuk-heading-s--file-type">File type</h3>
+                            <p class="govuk-body govuk-body--file-type">(e.g. docx, pdf, tiff)</p>
                         </div>
-                    </div>
-                    <div class="filters-form__consignment-ref__container">
-                        <p class="govuk-body browse__body">
-                            Consignment <abbr title="reference">ref</abbr>
-                        </p>
-                        <div class="govuk-form-group filters-form__group filters-form__consignment-ref-group">
-                            <label class="govuk-label" for="browse_filter_consignment-ref"></label>
-                            <input class="govuk-input filters-form__consignment-ref--input"
-                                   id="browse_filter_consignment-ref"
-                                   name="width10"
-                                   type="text">
+                        <div class="govuk-form-group govuk-form-group--consignment-filter">
+                            <input class="govuk-input govuk-!-width-full govuk-input--consignment-input"
+                                   id="full"
+                                   name="file_type"
```

Comment:
did we not agree to remove file type?

Reply:
we have to work on separate ticket for it as it is not part of current sprint

Reply:
file type has been removed from the consignment view as a filter - updated query, test cases and removed file type references from every where